features/Notes-content/Notes-content.js

```diff
@@ -0,0 +1,176 @@
+document.addEventListener('DOMContentLoaded', () => {
+    // Sections for adding and displaying notes
+    const addCardSection = document.getElementById('add-card');
+    const displayCardsSection = document.getElementById('display-cards');
+    const popup = document.getElementById('popup');
+
+    // Dummy data for initial notes
+    let notes = [
+        {
+            title: "Title 1",
+            content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam faucibus arcu eros, sit amet pulvinar leo bibendum nec. Duis fermentum ut ligula eu varius. Integer nec ornare arcu. Sed blandit lobortis consectetur. Etiam non lacinia sem, nec porta leo. Nam sit amet nulla congue, efficitur risus in, pellentesque nibh. Maecenas scelerisque lacus nunc, a rutrum est laoreet et. Mauris id commodo nunc. Vestibulum sit amet tristique lectus, eget malesuada risus. Proin facilisis odio et nisl congue, eget facilisis augue commodo. Maecenas lobortis enim vel quam pulvinar, quis sagittis mauris consectetur. Duis imperdiet magna lacus, sit amet convallis tortor interdum in. Suspendisse aliquam at lorem eu facilisis. Phasellus consequat nulla eget erat suscipit facilisis. Donec elementum, nisl placerat pulvinar interdum, sem justo sagittis est, ac fermentum orci mauris sed elit."
+        },
+        { title: "Title 2", content: "Quisque convallis ornare urna eu tincidunt. ".repeat(10) },
+        { title: "Title 3", content: "Curabitur sit amet maximus massa. ".repeat(10) },
+        { title: "Title 4", content: "Ut ultrices malesuada magna. ".repeat(3) },
+    ];
+
+    // Current note being edited
+    let note = { title: '', content: '' };
+
+    // Function to create the note editor in collapsed state
+    const addNote = () => {
+        addCardSection.innerHTML = `
+            <article class="add-card new-note">
+                <textarea
+                    id="content"
+                    class="card-details"
+                    placeholder="Take a note..."
+                    rows="1"
+                    onfocus="expandEditor()"
+                    oninput="updateContent(event)"
+                    onkeydown="handleEnter(event)"
+                >${note.content}</textarea>
+        `;
+    };
+
+    // Function to expand the note editor
+    window.expandEditor = () => {
+        const textarea = document.getElementById('content');
```

Comment:
Please use dialog instead this is it's main usage. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog